### DISCUSSION TITLE
Fullname column in datatable not working

### DISCUSSION THREAD

**Discussion by fftfaisal** [Power-Components/livewire-powergrid#1032] [General]
I am using the latest version of this package and using the Bootstrap theme.

In the database, I have first_name and last_name columns. So  I added the following code in the `addColumns() `Method

```
PowerGrid::eloquent()
    ->addColumn('first_name')
    ->addColumn('last_name')
    ->addColumn('full_name', fn (User $model) => e($model->fullName()))
```

in the `columns() ` method, I added following

`Column::make('Name', 'full_name' )->sortable()->searchable()`

in the `filters()` method, I added following

```
Filter::inputText('full_name')
                ->operators(['contains'])
                ->builder(function (Builder $query, $value) {
                    $searchValue = is_array($value) ? $value['value'] : $value;
                    return $query->where(\DB::raw('CONCAT(first_name, " ", last_name)'), 'like', "%{$searchValue}%" );
                }),
```
value displaying is ok, search filter is ok. but when trying to sort it is not working. I am getting the following SQL error. 

`SQLSTATE[42S22]: Column not found: 1054 Unknown column 'full_name' in 'order clause' `

I am not sure how to do it. any suggestion is welcome.

**Comment by Fludem** [comment]
Whenever the column name is different from the one in the database, remember to reference it in dataField in the [Column::field()](https://v4.livewire-powergrid.com/table/include-columns?id=fieldstring-field-string-datafield) method otherwise sortable will not work.

This isn't much help because I'm a ape not a developer, but it uses DB to sort, so you might need to make database field name 'first_name'

**Reply by MGeurts** [reply]
The above link is not working.

**Reply by luanfreitasdev** [reply]
https://livewire-powergrid.com/table/include-columns.html#field
